### DISCUSSION TITLE
小狼毫自动更新安装过程报病毒威胁

### DISCUSSION THREAD

**Discussion by heiher** [rime/home#1311] [General]
Windows 10系统弹出更新提醒，自动下载、安装过程中Windows Defender报安装程序存在病毒威胁。不知道是误报，还是发布构建环境被感染了。

![1](https://github.com/rime/home/assets/1407733/5f19b524-26f1-436a-b770-cbf0ae46eb3f)

**Comment by falsfe** [comment]
我是win11系统也报毒了
![图片](https://github.com/rime/home/assets/136047249/ff2cca7f-8ced-4926-bc98-15d76ac283db)
![图片](https://github.com/rime/home/assets/136047249/b7eeb4a3-2660-4d67-aa8c-4f19702f61ad)
而且阻止隔离和重装之后使用还是会有问题

**Comment by mirtlecn** [comment]
关联 https://github.com/rime/weasel/issues/914
